src/raydium/infos.ts

```diff
@@ -11,29 +11,35 @@ import { POOL_PROGRAM_ID_V4, FARM_PROGRAM_ID_V3, FARM_PROGRAM_ID_V5 } from "./id
 import { POOL_LAYOUT_V4, FARMER_LAYOUT_V3_2, FARMER_LAYOUT_V5_2, FARM_LAYOUT_V3, FARM_LAYOUT_V5 } from "./layouts";
 import { MARKET_STATE_LAYOUT_V3, _OPEN_ORDERS_LAYOUT_V2 } from "@project-serum/serum/lib/market";
 import BN from "bn.js";
-import { IPoolInfoWrapper, IFarmInfoWrapper, IInstancePool, IInstanceFarm, PoolDirection } from "../types";
+import { IPoolInfoWrapper, IFarmInfoWrapper, IInstancePool, IInstanceFarm, PoolDirection, PageConfig } from "../types";
 import { getBigNumber, getTokenAccount, TokenAmount } from "./utils";
 import { AccountLayout, MintLayout, RawAccount, RawMint } from "@solana/spl-token-v2";
 import { FarmerInfo, FarmInfo, PoolInfo } from ".";
 
 let infos: IInstancePool & IInstanceFarm;
 
 infos = class InstanceRaydium {
-  static async getAllPools(connection: Connection): Promise<PoolInfo[]> {
+  static async getAllPools(connection: Connection, page?: PageConfig): Promise<PoolInfo[]> {
     let pools: PoolInfo[] = [];
     //V4 pools
     const v4SizeFilter: DataSizeFilter = {
       dataSize: 752,
     };
     const v4Filters = [v4SizeFilter];
     const v4config: GetProgramAccountsConfig = { filters: v4Filters };
-    const allV4AMMAccount = await connection.getProgramAccounts(POOL_PROGRAM_ID_V4, v4config);
+    let allV4AMMAccount = await connection.getProgramAccounts(POOL_PROGRAM_ID_V4, v4config);
 
     let tokenAccountKeys: PublicKey[] = [];
     let mintAccountKeys: PublicKey[] = [];
     let openOrderKeys: PublicKey[] = [];
     let serumMarketKeys: PublicKey[] = [];
-
+    allV4AMMAccount.sort((a, b) => a.pubkey.toBase58().localeCompare(b.pubkey.toBase58()));
```

Comment:
1. Perhaps this piece of logic can be extracted to utils as a function? Ex: `Paginate`
2. Assign the batched accounts to a new variable (Ex: `pagedPoolAccounts`) instead of mutating `allV4AMMAccount`